scipy/spatial/src/distance_metrics.h

```diff
@@ -215,6 +516,80 @@ struct EuclideanDistance {
     }
 };
 
+struct JensenshannonDistance {
+    template <typename T>
+    struct Acc {
+        Acc(): js_diverg(0) {}
+        T js_diverg;
```

Comment:
I'll point out that the existing metrics that have an `Acc` struct only use it because there are multiple values being accumulated, not as a requirement. If you just wanted to name the value then that's fine though.